src/cc_catalog_airflow/Dockerfile

```diff
@@ -20,5 +20,6 @@ ADD ./dags ${AIRFLOW_HOME}/dags
 RUN pip install --user -r ${REQUIREMENTS_FILE}
 
 CMD python wait_for_db.py && \
-  airflow initdb && \
+  airflow db init && \
+  airflow users create -r Admin -u airflow -f Air -l Flow -p airflow --email airflow@example.org && \
```

Comment:
Peculiar choice of credentials!

Reply:
The longer version of the flags can make it easier to read, i.e. `--role` instead of `-r` or `--firstname` instead of `-f`, but this is just me nitpicking.

In general looks good to me, as it solves the warning errors.